index.html

```diff
@@ -108,6 +108,39 @@ <h2>A project for first-time contributions</h2>
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
 
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
edit `<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` at the start of your card
change `TEMPLATE: MAKE A COPY` with your name